api/preprint_providers/serializers.py

```diff
@@ -0,0 +1,34 @@
+from rest_framework import serializers as ser
+
+from api.base.utils import absolute_reverse
+from api.base.serializers import JSONAPISerializer, LinksField
+
+
+class PreprintProviderSerializer(JSONAPISerializer):
+
+    filterable_fields = frozenset([
+        'name',
+        'description',
+        'id'
```

Comment:
`id`'s shouldn't really be filterable fields - that's what a detail view is for - unless there's strong use case for it. Does `ember-preprints` need this?